packages/react-components/src/components/chart/converters/padYAxis.ts

```diff
@@ -0,0 +1,150 @@
+import { DataPoint, DataStream, Threshold } from '@iot-app-kit/core';
```

Comment:
note for other reviewers, this is pulled directly from the synchro-charts codebase :) reduce reuse recycle